samples/nrf9160/modem_shell/src/uart/uart_shell.c

```diff
@@ -103,24 +108,11 @@ void uart_toggle_power_state(void)
 		/* allow little time for printing the notification */
 		k_sleep(K_MSEC(500));
 
-		/* set uart0 to suspended state */
-		err = pm_device_action_run(uart_dev, PM_DEVICE_ACTION_SUSPEND);
-
-		/* set uart1 to suspended state */
-		uart_dev = device_get_binding(DT_LABEL(DT_NODELABEL(uart1)));
-		if (uart_dev) {
-			pm_device_action_run(uart_dev, PM_DEVICE_ACTION_SUSPEND);
-		}
+		disable_uarts();
 	} else {
-		/* set uart0 to active state */
-		err = pm_device_action_run(uart_dev, PM_DEVICE_ACTION_RESUME);
-
-		/* set uart1 to active state */
-		uart_dev = device_get_binding(DT_LABEL(DT_NODELABEL(uart1)));
-		if (uart_dev) {
-			pm_device_action_run(uart_dev, PM_DEVICE_ACTION_RESUME);
-		}
-		mosh_print("Enabling UARTs");
+		enable_uarts();
+
+		mosh_print("Enabled UARTs");
```

Comment:
The words could be reordered to be in line with similar text on line 44. There should be no test case for this yet, so this can be changed without affecting tests.
```suggestion
		mosh_print("UARTs enabled");
```

Reply:
alright. will do.